packages/appstore/src/components/withdraw-request-verification/withdraw-request-verification.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import WithdrawalVerificationEmail from '@deriv/cashier/src/pages/withdrawal/withdrawal-verification-email';
+
+const WithdrawRequestVerification = () => {
+    return <WithdrawalVerificationEmail />;
+};
```

Comment:
@hamid-deriv This component seems to be redundant, We will be adding things to it later?

Reply:
This component will be used for the content of withdraw tab. I renamed it so it's more descriptive now 👍